### PR TITLE
RAC-3778 - re-factor log formtting

### PR DESCRIPTION
**Pull request by larry-dean**
* This commit  also includes changes for RAC-3776
* Place process, path, and greenlet informaton at end of trace line as suffix
* If 1st character of message is ' +-', don't include suffix information
* Give each test a indentification number
* Remove addtional lines for test block start, test start and test end messages
* Created new simple format and rename origial format to 'original'
* Start block and test ID at 1 instead of 0

### CODE REVIEW

**Comment by larry-dean** [comment]
@stuart-stanley @hohene @derrickostertag @gpaulos @RackHD/corecommitters

File: test/stream-monitor/flogging/infra_logging.py [hunk 1]
```diff
@@ -46,8 +46,23 @@ def filter(self, record):
...
+        # This section removes the process, path, and greenlet information 
+        # from any mssage that starts with any of the characters ' +-'. 
+        # These are well known stream monitor messages and this informtion
+        # is not required to be displayed.
+        # TODO: there is probably a better way to do this but this will
+        #       work until it is setup.
+        if any(elem in record.msg[0] for elem in r' +-?%'):
+            record.location_info_string = ''
+        else:    
+            record.location_info_string = '{} {} {}:{}@{} {}'.format(record.process,
```
Comment (stuart-stanley) [inline test/stream-monitor/flogging/infra_logging.py]: Go ahead and remove 'original' from below, but move the "%(process)d  %(processName)s %(pathname)s:%(funcName)s@%(lineno)d %(greenlet)s'" here. Err, that might be clearer if I just showed it :):

`                record.location_info_string = '%(process)d  %(processName)s %(pathname)s:%(funcName)s@%(lineno)d %(greenlet)s'.format(record)`

That keeps the connection between the logging formatters and this statement much more clear, I think.

Comment (larry-dean) [inline test/stream-monitor/flogging/infra_logging.py]: Will do!

**Comment by larry-dean** [comment]
Changes suggest by @stuart-stanley pushed up

**Comment by larry-dean** [comment]
Correction to issues found by @hohene pushed up

**Comment by johren** [comment]
test this please

**Pull request merged**